server/index.js

```diff
@@ -15,4 +18,25 @@ app.get("/api", (req, res) => {
   res.send("Hi there");
 });
 
-app.listen(3000, () => console.log("Now listening on port 3000!"));
+//Get all items
+app.get('/api/items', (req, res) => {
+  db.getAll()
+  .then( results => {res.send(results)})
+});
+
+
+//Seed items database
+app.post('/api/items/seed', (req, res) => {
+
+  console.log(req.body.data)
+  console.log(req.body.data.constructor)
+
+  Promise.all(req.body.data.map(jsonItem => {
```

Comment:
Not sure if you're actually going to use this in the real thing, but you can abstract some of this with
[`Model.insertMany()`](https://mongoosejs.com/docs/api/model.html#model_Model.insertManyl)

Reply:
Nice! I just did that and it saved a lot of code.